docs/metrics.md

```diff
@@ -2,14 +2,16 @@
 
 When [Caddy metrics](https://caddyserver.com/docs/metrics) are enabled, FrankenPHP exposes the following metrics:
 
-* `frankenphp_[worker]_total_workers`: The total number of workers.
-* `frankenphp_[worker]_busy_workers`: The number of workers currently processing a request.
-* `frankenphp_[worker]_worker_request_time`: The time spent processing requests by all workers.
-* `frankenphp_[worker]_worker_request_count`: The number of requests processed by all workers.
-* `frankenphp_[worker]_ready_workers`: The number of workers that have called `frankenphp_handle_request` at least once.
-* `frankenphp_[worker]_worker_crashes`: The number of times a worker has unexpectedly terminated.
-* `frankenphp_[worker]_worker_restarts`: The number of times a worker has been deliberately restarted.
 * `frankenphp_total_threads`: The total number of PHP threads.
 * `frankenphp_busy_threads`: The number of PHP threads currently processing a request (running workers always consume a thread).
+* `frankenphp_queue_depth`: The number of regular queued requests
+* `frankenphp_total_workers{worker="[worker_name]"}`: The total number of workers.
+* `frankenphp_busy_workers{worker="[worker_name]"}`: The number of workers currently processing a request.
+* `frankenphp_worker_request_time{worker="[worker_name]"}`: The time spent processing requests by all workers.
+* `frankenphp_worker_request_count{worker="[worker_name]"}`: The number of requests processed by all workers.
+* `frankenphp_ready_workers{worker="[worker_name]"}`: The number of workers that have called `frankenphp_handle_request` at least once.
+* `frankenphp_worker_crashes{worker="[worker_name]"}`: The number of times a worker has unexpectedly terminated.
+* `frankenphp_worker_restarts{worker="[worker_name]"}`: The number of times a worker has been deliberately restarted.
+* `frankenphp_worker_queue_depth{worker="[worker_name]"}`: The number of queued requests.
 
-For worker metrics, the `[worker]` placeholder is replaced by the worker script path in the Caddyfile.
+For worker metrics, the `[worker_name]` placeholder is replaced by the worker name in the Caddyfile.
```

Comment:
We should also explain what will be the placeholder if the name is not set explicitly.